### DISCUSSION TITLE
User impersonation has disappeared in Evoq v. 09.06.04

### DISCUSSION THREAD

**Discussion by devray90** [dnnsoftware/Dnn.Platform#4395] [Q&A]
Hi guys,

User impersonation "loginAs" button is now missing. Can someone please confirm if this has been moved, or removed? And if it has been removed, why? It was a very useful feature for viewing diagnosing issues from the user perspective.

![image](https://user-images.githubusercontent.com/59833246/103546514-01add900-4e9b-11eb-960f-6ff07b7af7ad.png)

**Comment by bdukes** [comment]
You'll need to open a ticket with [Evoq support](https://dnnsupport.dnnsoftware.com/), this repository is only for DNN Platform (which does not include this feature).
